### PR TITLE
Pull request for basic destroy friendship

### PR DESCRIPTION
**Pull request by AlanJager**

### CODE REVIEW

**Comment by AlanJager** [comment]
I just find where to do the request but not quiet sure about how to make it work. Could you show me the basic procedure of adding a function?

**Comment by AlanJager** [comment]
I have tried to add the destroyFriendship function into container and than invoke it but not a function error occurred.

**Comment by bokuweb** [comment]
If you want to call `destroyFriendship`, please dispatch `Action` from `container`.

e.g. https://github.com/bokuweb/Tsukiakari/blob/master/src/renderer/src/containers/tweetitem-footer.js#L22

Above code is example of dispatch retweet / favaorite action from container.
If you have dispatched action, please receive action by `saga`. 
(`saga` is redux-saga feature.)

e.g. https://github.com/bokuweb/Tsukiakari/blob/master/src/renderer/src/sagas/tweets.js#L48-L58

Above code is example of call destroyRetweet.

**Comment by bokuweb** [comment]
@AlanJager merged 🎉

File: src/renderer/src/components/account-tooltip.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
 import React, { Component, PropTypes } from 'react';
 import { Button } from 're-bulma';
 import B from '../lib/bem';
+import T from '../lib/twitter-client';
```
Comment (bokuweb) [inline src/renderer/src/components/account-tooltip.js:4]: oops.., sorry, I had merged..
 Please remove this line, if you will request next PR.
we have not to call twitter client API from component.
we should call api from `sagas`.

Comment (AlanJager) [reply src/renderer/src/components/account-tooltip.js:4]: Oh, I'm sorry for this miss, I think I forgot remove this line

**Pull request merged**